packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -263,7 +263,8 @@ export abstract class Chart extends Observable {
 
         this.tooltip = new Tooltip();
         this.seriesLayerManager = new SeriesLayerManager(this.seriesRoot, this.highlightRoot, this.annotationRoot);
-        const ctx = (this.ctx = new ChartContext(this, {
+        const mode: ChartMode = (options.userOptions as { mode?: ChartMode }).mode ?? this.mode;
+        const ctx = (this.ctx = new ChartContext(this, mode, {
```

Comment:
> [!WARNING]
> This change is not type-safe, but I cannot find a better way to retrieve the `mode` option because the `Chart.mode` property is still set to its default `'standalone'` value at this point. Open to better suggestions.